new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiUpdateDomainConfig.java

```diff
@@ -81,21 +84,55 @@
 import static oracle.weblogic.kubernetes.assertions.TestAssertions.operatorIsReady;
 import static oracle.weblogic.kubernetes.assertions.TestAssertions.podDoesNotExist;
 import static oracle.weblogic.kubernetes.assertions.TestAssertions.verifyRollingRestartOccurred;
+import static oracle.weblogic.kubernetes.utils.CommonPatchTestUtils.patchDomainWithNewSecretAndVerify;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkPodReady;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.checkServiceExists;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createConfigMapAndVerify;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.createSecretWithUsernamePassword;
 import static oracle.weblogic.kubernetes.utils.CommonTestUtils.getPodCreationTime;
+import static oracle.weblogic.kubernetes.utils.CommonTestUtils.verifyCredentials;
 import static oracle.weblogic.kubernetes.utils.ExecCommand.exec;
 import static oracle.weblogic.kubernetes.utils.ThreadSafeLogger.getLogger;
 import static org.awaitility.Awaitility.with;
 import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
+/**
+ * This test class verifies the following scenerios
+ *
+ * <p>testMiiCheckSystemResources
+ *  Check the SystemResourece in a pre-configured in ConfigMap
+ *
+ * <p>testMiiDeleteSystemResources
+ *  Delete SystemResourece(s) defined in WebLogic domain 
+ *
+ * <p>testMiiAddSystemResources
+ *  Add new SystemResources to a running domain
+ *
+ * <p>testMiiAddDynmicClusteriWithNoReplica
+ *  Add a new dynamic WebLogic cluster to a running domain with default Replica
+ *  count(zero), so that no manged server on the new cluster is activated.
+ *
+ * <p>testMiiAddDynamicCluster
+ *  Add a new dynamic WebLogic cluster to a running domain with non-zero Replica
+ *  count so that required number of manged servers(s) on new cluster get  
+ *  activted after rolling restart. 
+ *
+ * <p>testMiiAddConfiguredCluster
+ *  Add a new configured WebLogic cluster to a running domain 
+ *
+ * <p>testMiiUpdateWebLogicCredential
+ *  Update the adminstrative credential of a running domain by updating the 
+ *  secret and activating a rolling restart.
+ */
+
 @TestMethodOrder(MethodOrderer.OrderAnnotation.class)
-@DisplayName("Test to add a cluster to model in image domain")
+@DisplayName("Test add SystemResources, Clusters to model in image domain")
 @IntegrationTest
-class ItMiiAddCluster {
+class ItMiiUpdateDomainConfig {
```

Comment:
remove old code and use installAndVerifyOperator utility method in initAll()

Reply:
Added installAndVerifyOperator to replace the old code.